README.md

```diff
@@ -1,5 +1,7 @@
 # React In-depth: An exploration of UI development
-At [DevelopmentArc&reg;](http://developmentarc.com) we have a long history with UI application development. We started back in the Authorware/Toolbook days, then moved to Director, then HTML 1.0, moved over to Prototype.js and at the same time Flash. With the release of Flex, we jumped in feet first and then dug deeper into AIR and device development. With the collapse of Flash/Flex we found ourselves back in HTML and JavaScript.
+At [DevelopmentArc&reg;](http://developmentarc.com) we have a love affair with UI application development and the technologies and frameworks that help us manage their complexity and priority in the application stack. 
```

Comment:
How about:

At [DevelopmentArc&reg;](http://developmentarc.com) we have a love affair with UI application development. We enjoy the exploration of technologies and frameworks, which help us manage the end-goals and complexity of the application stack.